orchestration/hca_orchestration/tests/e2e/conftest.py

```diff
@@ -1,133 +1,7 @@
-import logging
```

Comment:
Sorry, yeah Andrea was also confused by this - I should have been more explicit in the readme.
I decided to disable all of the e2e tests because TDR dev become unresponsive, they timeout and then the check fails and that whole process takes ~1 hour. Whereas I can run a smoke test against TDR prod in ~15 minutes manually. 
So, for now, until I refactor the e2e tests to run against TDR prod [FE-204](https://broadworkbench.atlassian.net/browse/FE-204), I'm skipping all of the tests in the e2e folder.

To accomplish this, I first renamed the original conftest.py to conftest_orig.py (which is why it looks like it was cleared out), as I will want this once the refactor is done. I then create a new conftest.py, which essentially adds the skip decorator to each test. And that is why line 15 in test_load_hca.py is removed. I don't need to hardcode the skip for that one separately, as they are all marked dynamically.

If that all makes better sense, I will add it to the PR description.

Apologies for the confusion!

Reply:
See below - after discussion we agreed that not running the e2e tests during PR validation was the better solution.